plugins/modules/mso_schema_template_anp_epg_useg_attribute.py

```diff
@@ -229,11 +229,13 @@ def main():
         payload = dict(name=name, displayName=name, description=description, type=EPG_U_SEG_ATTR_TYPE_MAP[attribute_type], value=value)
 
         if attribute_type == "ip":
-            if useg_subnet is False:
+            if useg_subnet:
+                if value != "" and value != "0.0.0.0" and value is not None:
+                    mso.fail_json(msg="The value should be empty string or 0.0.0.0, when the useg_subnet is true.")
```

Comment:
mso.fail_json(msg="The value of uSeg should be an empty string or 0.0.0.0, when the useg_subnet is set to true.")